docs/organizer-guide/confs/website.rst

```diff
@@ -0,0 +1,293 @@
+.. _conf-web-resources:
+
+.. highlight:: yaml
+
+Conference websites
+===================
+
+Each conference has a section of the website, which is part of the same
+repository as the main conference website. Many parts of the conference
```

Comment:
The sentence currently reads: "Each conference has a section of the website, which is part of the same repository as the main conference website."

Usage of the word "conference" confuses me, so I went with a suggestion based on what seemed logical in my head. Our www repository has the main community website, and a directory for each conference, so that's what seemed to make sense. I fixed the typo ;)